File: Java/gesbrasilburger/src/main/java/sn/brasilburger/entity/Zone.java
package sn.brasilburger.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Zone {
    private int id;
    private String nom;
    private Double prixLivraison;
    private boolean etat = true;
    public String toString(){
        return " Complement->[ID: " + id + ", Nom: " + nom + ", Prix Livraison: " + prixLivraison + "]";
    }
}

File: CSharp/brasilBurger/Models/Zone.cs
namespace brasilBurger.Models
{
    public class Zone
    {
        public int Id { get; set; }
        public string Nom { get; set; }
        public decimal PrixLivraison { get; set; }
        public bool Etat { get; set; }
        public ICollection<Livreur> Livreurs { get; set; } = new List<Livreur>();
        public ICollection<Commande> Commandes { get; set; } = new List<Commande>();
    }
}